sdf/1.11/particle_emitter.sdf

```diff
@@ -26,8 +26,6 @@
     depending on the emmiter type:
       - point: The area is ignored.
       - box: The area is interpreted as width X height X depth.
-      - cone: The area is interpreted as the bounding box of the
```

Comment:
We need to remove it from 1.12 as well.

Reply:
ah yes, removed. [ca11244](https://github.com/gazebosim/sdformat/pull/1434/commits/ca11244a3cd089bfd70d9a111735d025142ac723)